XSUP-53329 (#40771)
demisto/content: Packs/CrowdStrikeFalconSandbox/Integrations/CrowdStrikeFalconSandboxV2/README.md

## Additional Notes

- The CrowdStrike Environment ID 300 has been deprecated and is now EOL. It is recommended to use Environment ID 310 for Linux sandboxing.

### cs-falcon-sandbox-report-summary

***

> Gets summary information for a job_id or sha256

Reviewer: ```suggestion
Retrieves summary information for the job_id or the sha256.
```

#### Base Command

`cs-falcon-sandbox-report-summary`

#### Input

| **Argument Name** | **Description** | **Required** |
| --- | --- | --- |

> | file | A comma-separated list of file hashes (SHA256), followed by an environment ID. Each pair is separated by a colon (:), eg: SHA256_1:ENV_ID_1,SHA256_2:ENV_ID_2. Provide either this argument or job_id. | Optional |

Reviewer: See my comments in the yml file
